www/main/settings/components/SettingViewer.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { useStoreState } from 'easy-peasy';
+
+const SettingViewer = () => {
```

Comment:
Think the new window is causing some compactness. 
![image](https://user-images.githubusercontent.com/2982784/110659637-9bce1e00-8177-11eb-8273-7453efef0b08.png)

Reply:
@tsingh777 @maneetpaul we need to figure out what's our min supported resolution. I am facing similar problems with new designs for search pane as well.